client/pom.xml

```diff
@@ -58,5 +58,10 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I don't think you need this dependency here. You didn't add anything to the client.